### PR TITLE
Prettier Loading Screens

### LINKED ISSUE #8279: Editor: Add loading screen
## Feature Description

When the editor is loading is a primitive loading screen is displayed, showing the words, "Please wait..." with a black background. 

Screenshot 2021-07-07 at 15 26 25

Styling on this loading page should improved, not only to make it look a little nicer, but also adds some sort of loading animation / progress bar ( that does not need to be actuate ), to give the user some feeling like that "something" is happening, to improve user's perception of speed of the application. 

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### LINKED ISSUE #8280: Dashboard: Add loading screen
## Feature Description

When the dashboard is loading is a primitive loading screen is displayed, showing the words, "Please wait..." with a white background.

Screenshot 2021-07-07 at 15 27 56

Styling on this loading page should improved, not only to make it look a little nicer, but also adds some sort of loading animation / progress bar ( that does not need to be actuate ), to give the user some feeling like that "something" is happening, to improve user's perception of speed of the application.

Follow up from https://github.com/google/web-stories-wp/issues/8279

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by BrittanyIRL**
## Context

While discussing perf wins there were some laments about the loading screen in WordPress while the app loads. This just adds a spinner and makes the screen a little easier on the eyes. 

## Relevant Technical Choices

- Just made a spinner with css, we don't have access to the app yet so can't reuse the existing spinner. This one isn't as slick as the one in the design system, but it's entirely css and adds no new elements to the page. 

## To-do

- I don't think any adjustments are needed for a11y but figure I'll call it out here in case I'm wrong :) 

## User-facing changes

| Before | After | 
| --- | --- |
| Screen Shot 2021-07-14 at 1 50 54 PM | dashboard |

| Before | After | 
| --- | --- |
| Screen Shot 2021-07-15 at 10 48 59 AM | editor |

## Testing Instructions

just load the dashboard and editor, should see update matching screenshots.

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #8280 
Fixes #8279

### CODE REVIEW

File: assets/src/dashboard/style.css [hunk 1]
```diff
@@ -51,7 +51,81 @@ body.web-story_page_stories-dashboard #wpbody {
...
+  top: 0;
+  right: 0;
+  bottom: 0;
+  margin: 20% auto auto;
+  width: 0.4em;
+  height: 0.4em;
+  border-radius: 50%;
+
+  animation: spin-progress 0.85s infinite
+    cubic-bezier(0.455, 0.03, 0.515, 0.955);
```
Comment (BrittanyIRL) [inline assets/src/dashboard/style.css:129]: timing and ease copied from design system spinner implementation

**Review by dreamofabear — APPROVED** [review]
Looks nice!

**Comment by spacedmonkey** [comment]
Screenshot 2021-07-15 at 13 44 46

Positioning of the please wait if the browser does have a lot of height.

**Review by maxyinger — APPROVED** [review]

**Comment by BrittanyIRL** [comment]
> Positioning of the please wait if the browser does have a lot of height.

Fair, updated to use px instead of percentage and fix the position towards the top.

**Review by spacedmonkey — DISMISSED** [review]
This looks great. @BrittanyIRL One nit pic. The "please wait" text should font family of `'Google Sans',sans-serif` so it matches the editor / dashboard.

**Pull request merged**